qiskit/providers/backend_compat.py

```diff
@@ -119,29 +121,43 @@ def convert_to_target(
                 params=params,
             )
         else:
-            logger.warning(
-                "Definition of instruction %s is not found in the Qiskit namespace and "
-                "GateConfig is not provided by the BackendConfiguration payload. "
-                "Qiskit Gate model cannot be instantiated for this instruction and "
-                "this instruction is silently excluded from the Target. "
-                "Please add new gate class to Qiskit or provide GateConfig for this name.",
-                name,
+            warnings.warn(
+                f"No gate definition for {name} can be found and is being excluded "
+                "from the generated target. You can use `custom_name_mapping` to provide "
+                "a definition for this operation.",
+                RuntimeWarning,
             )
             unsupported_instructions.append(name)
 
     for name in unsupported_instructions:
         all_instructions.remove(name)
 
-    # Create empty inst properties from gate configs
-    for name, spec in gate_configs.items():
-        if hasattr(spec, "coupling_map"):
-            coupling_map = spec.coupling_map
-            prop_name_map[name] = dict.fromkeys(map(tuple, coupling_map))
-        else:
+    # Create inst properties placeholder
+    for name in all_instructions:
+        if name not in prop_name_map:
```

Comment:
Can this ever be `True`? `all_instructions` is a set so there aren't any duplicates and `prop_name_map` on the first iteration is empty.

Reply:
You're right. Updated initialization of `prop_name_map`